jobs/suite/default.env

```diff
@@ -1,2 +1 @@
-GINKGO_TEST_ARGS=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]
 GINKGO_PARALLEL=y
```

Comment:
AKA this is not part of migrating e2e-runner.sh env to kubetest but the opposite (migrating ginkgo-e2e.sh stuff into kubetest)